js/main.js

```diff
@@ -1,7 +1,46 @@
-
 // so that we can use anywhere, will be used for pretty much everything
 let storage = window.localStorage;
 
+document.addEventListener("DOMContentLoaded", function() {
+  loadTodos();
+
+  if (isUsernameSet()) {
+    loadUsername()
+  } else {
+    let usernameElement = document.querySelector(".username h1 span")
+    usernameElement.textContent = "User"
+  }
+});
+
+let loadTodos = () => {
+  let todos = []
+  /* get all the items and remove the ones
+  ** that are not todos
+  */ 
+  for(let i = 0; i < storage.length; i++) {
+    if(storage.key(i).substr(0, 1) == '_') {
```

Comment:
Good practice is to always use "===" instead of "==".  Then you check the type and value, here it works either way but just good practice 👍

Reply:
👍 👍